src/modules/feature/auto-logout/AutoLogoutWarning.vue

```diff
@@ -119,7 +120,7 @@ watch(
 		width: 100%;
 		font-size: var(--text-lg);
 
-		@include breakpoint(tablet) {
+		@media #{map.get($display-breakpoints, 'sm-and-up')} {
```

Comment:
```suggestion
		@media #{map.get($display-breakpoints, 'md-and-up')} {
```
Otherwise it gets too narrow and doesn't look good.